src/hub.ts

```diff
@@ -63,6 +64,15 @@ export class Hub extends EventEmitter {
     }
 
 
+    /**
+     * @readonly
+     * @property {string} macAddress Primary mac address of the hub
+     */
+    public get macAddress () {
```

Comment:
Technically speaking this is not a MAC address, it is a Bluetooth address (they just happen to have the same format).

Reply:
As the LWP refers to it as "Primary MAC Address", I was going for the same terminology but maybe simply "address" would be more technically accurate.

Reply:
Most people will never know the difference anyway :smile:

Reply:
Small niggle triggering my OCD - can we standardise on "MAC" address (uppercase), as used below? :)

Reply:
Regarding MAC address vs just address, I guess it makes most sense to follow Lego's own terminology, since this is reportable only through their protocol. I guess it would cause less confusion if it was consistent?

Reply:
Then I would go for `primaryMACAddress` as secondary one exists in the protocol. What do you think ?

Reply:
I like it, that sounds good to me!